### DISCUSSION TITLE
Can I use this to make FNF-style songs?

### DISCUSSION THREAD

**Discussion by guest123guest** [stepmania/stepmania#2239] [Q&A]
Friday Night Funkin is a stupid game, and all of the engines I found for making "mods" for it are either dead or don't compile properly with the shitty haxe tools. I went here since I believe Stepmania has similar controls but is also a better engine overall despite also being mostly dead. Is there any modes on here that allow me to make FNF-style songs?

**Comment by geefr** [comment]
https://www.noti.tg/
